### PR TITLE
Add commands to test development/kops/install_requirements.sh

### PR DESCRIPTION
**Pull request by rcrozean**
*Issue #, if available:*

*Description of changes:*
Add make targets for testing the files in `development/kops/*.sh` like [development/kops/install_requirements.sh](https://github.com/aws/eks-distro/blob/main/development/kops/install_requirements.sh). These files are important to our build tasks and should be tested validity when changed.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: development/kops/run_all.sh [hunk 1]
```diff
@@ -17,44 +17,46 @@ set -exo pipefail
...
-      createDate=$(date -d"$createDate" +%s)
-      olderThan=$(date --date "1 day ago" +%s)
-      if [[ $createDate -lt $olderThan ]]; then
-        echo "Deleting cluster ${cluster_fqdn}"
-        ${KOPS} delete cluster --state "${KOPS_STATE_STORE}" --name ${cluster_fqdn} --yes
-        aws s3 rm --recursive "${KOPS_STATE_STORE}/${cluster_name}"
-      fi
-    fi
-  done
+if [[ "${KOPS_STATE_STORE}" != "" && "$JOB_TYPE" != "presubmit" ]]; then
```
Comment (jaxesn) [inline development/kops/run_all.sh:36]: i might would reorder this file a bit if you can to avoid havng to check this twice.

File: Makefile [hunk 2]
```diff
@@ -104,7 +104,7 @@ kops-arm-ubuntu-22: kops-prereqs
 	RELEASE=$(RELEASE) $(KOPS_ENTRYPOINT);
 
 .PHONY: kops-prereqs
-kops-prereqs: postsubmit-build
+kops-prereqs: $(if $(JOB_TYPE) == "presubmit",,postsubmit-build)
```
Comment (jaxesn) [inline Makefile:107]: this is not quite right...  you probably want this

`$(if $(filter presubmit,$(JOB_TYPE)),,postsubmit-build)`

Check Common.mk youll see this kind of thing.

**Review by jaxesn — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (rcrozean) [reply Makefile:107]: Used [open_pr_check.sh](https://github.com/aws/eks-distro-build-tooling/blob/64e9741d152acddf6b6aa54cdbae77ac728232be/pr-scripts/open_pr_check.sh#L26) as the example. Will update to your recommendation. Don't see us doing this in eks-distro, other than a single call in the [buildkit.sh](https://github.com/aws/eks-distro-build-tooling/blob/64e9741d152acddf6b6aa54cdbae77ac728232be/scripts/buildkit.sh#L92) where we trim, but not sure why this is the only example of us trimming in that repo.

**Review by jaxesn — APPROVED** [review]

**Pull request merged**